designer/server/src/views/help/accessibility-statement.njk

```diff
@@ -0,0 +1,346 @@
+{% extends "layouts/page.njk" %}
+
+{% set pageTitle = "Accessibility statement" %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-xl">
+        Accessibility statement for [website name]
```

Comment:
I've used the [**Sample accessibility statement** (for a fictional public sector website)](https://www.gov.uk/government/publications/sample-accessibility-statement/sample-accessibility-statement-for-a-fictional-public-sector-website) for now replacing the [Home Office statement (see above)](https://github.com/DEFRA/forms-designer/pull/69/files#diff-dfe179dd204600870d7f43040d35d0a4f3164a967a2b496ab4f82cfede92eb0f) version